tests/test_compound_priority.py

```diff
@@ -0,0 +1,67 @@
+#  type: ignore
```

Comment:
you shouldn't have to `type: ignore` all the tests one by one since I already excluded the `tests` repository from mypy in the `pyproject.toml`

Reply:
yes but they are bugging me in the local linter and when i call `pre-commit run --all-files`

Reply:
did you `poetry run pre-commit run -a` ? So that it takes into account all the options